src/command.h

```diff
@@ -413,10 +415,33 @@ class ProbeSSBOCommand : public Probe {
   std::vector<Value> values_;
 };
 
+/// Base class for BufferCommand and SamplerCommand to handle binding.
+class BindableResourceCommand : public PipelineCommand {
+ public:
+  explicit BindableResourceCommand(Type type, Pipeline* pipeline);
```

Comment:
nit: only need `explicit` if there is a single parameter

Reply:
Explicit is not limited to a single parameter (see [example](https://en.cppreference.com/w/cpp/language/explicit)). Do we still want to remove this from BindableResourceCommand and BufferCommand constructors?

Reply:
Yea, we only mark explicit on constructors with a single parameter.

Reply:
Ok, done.